test/sequential/test-process-disable-warnings.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+const execFile = require('child_process').execFile;
+const warnmod = require.resolve(fixtures.path('disable-warnings.js'));
+const node = process.execPath;
+
+const dep0025Message = /\(node:\d+\) \[DEP0025\] DeprecationWarning: sys is deprecated\. Use util instead\./;
+const dep0094Message = /\(node:\d+\) \[DEP0094\] DeprecationWarning: assert\.fail\(\) with more than one argument is deprecated\. Please use assert\.strictEqual\(\) instead or only pass a message\./;
+const experimentalWarningMessage = /\(node:\d+\) ExperimentalWarning: vm\.measureMemory is an experimental feature and might change at any time/;
+
+// Show All Warnings
+execFile(node, [warnmod], function(er, stdout, stderr) {
+  assert.strictEqual(er, null);
+  assert.strictEqual(stdout, '');
+  assert.match(stderr, dep0025Message);
+  assert.match(stderr, dep0094Message);
+  assert.match(stderr, experimentalWarningMessage);
+});
+
+// Hide All Warnings
+execFile(node, ['--no-warnings', warnmod], function(er, stdout, stderr) {
+  assert.strictEqual(er, null);
+  assert.strictEqual(stdout, '');
+  assert.doesNotMatch(stderr, dep0025Message);
+  assert.doesNotMatch(stderr, dep0094Message);
+  assert.doesNotMatch(stderr, experimentalWarningMessage);
+});
+
+// Hide Only DEP0025 Warnings
+execFile(node, ['--disable-warnings=DEP0025', warnmod], function(er, stdout, stderr) {
+  assert.strictEqual(er, null);
+  assert.strictEqual(stdout, '');
+  assert.doesNotMatch(stderr, dep0025Message);
+  assert.match(stderr, dep0094Message);
+  assert.match(stderr, experimentalWarningMessage);
+});
+
+// Hide DEP0025 and DEP0094 Warnings
+execFile(node, ['--disable-warnings=DEP0025', '--disable-warnings=DEP0094', warnmod], function(er, stdout, stderr) {
+  assert.strictEqual(er, null);
+  assert.strictEqual(stdout, '');
+  assert.doesNotMatch(stderr, dep0025Message);
+  assert.doesNotMatch(stderr, dep0094Message);
+  assert.match(stderr, experimentalWarningMessage);
+});
+
+// Hide DeprecationWarnings
+execFile(node, ['--disable-warnings=DeprecationWarning', warnmod], function(er, stdout, stderr) {
+  assert.strictEqual(er, null);
+  assert.strictEqual(stdout, '');
+  assert.doesNotMatch(stderr, dep0025Message);
+  assert.doesNotMatch(stderr, dep0094Message);
+  assert.match(stderr, experimentalWarningMessage);
+});
+
+execFile(node, ['--disable-warnings=ExperimentalWarning', warnmod], function(er, stdout, stderr) {
+  assert.strictEqual(er, null);
+  assert.strictEqual(stdout, '');
+  assert.match(stderr, dep0025Message);
+  assert.match(stderr, dep0094Message);
+  assert.doesNotMatch(stderr, experimentalWarningMessage);
+});
```

Comment:
All of these callbacks need to be wrapped in `mustCall`; generally I think it's better to follow the pattern of the newer tests that use the test runner and `spawnPromisified` and are written in ESM.

Reply:
Great feedback, I'll make this changes asap. I copied an existing test for these, that is all.

Reply:
Do you have an example of this anywhere? I searched for `spawnPromisified` in the `test` directory but nothing is coming up.

Reply:
I get 283 results in 57 files for `spawnPromisified` 🤔 here's an example: https://github.com/nodejs/node/blob/052434a0c1f8b95ed5df488cb6484528135c8c4d/test/es-module/test-esm-extension-lookup-deprecation.mjs#L22-L29

Reply:
Lol totally my bad - I had the filter on `exclude` instead of `include`. Whoops!